pkg/client/imagerepositories.go

```diff
@@ -56,6 +61,13 @@ func (c *imageRepositories) Get(name string) (result *imageapi.ImageRepository,
 	return
 }
 
+// Get returns information about a particular imagerepository in a particular namespace and error if one occurs.
+func (c *imageRepositories) GetByNamespace(namespace, name string) (result *imageapi.ImageRepository, err error) {
+	result = &imageapi.ImageRepository{}
```

Comment:
you don't have to do this, the result should be initialized as it is return value.